torchvision/transforms/functional.py

```diff
@@ -82,6 +82,54 @@ def to_tensor(pic):
         return img
 
 
+def as_tensor(pic):
+    """Convert a ``PIL Image`` or ``numpy.ndarray`` to tensor of same type.
+
+    See ``AsTensor`` for more details.
+
+    Args:
+        pic (PIL Image or numpy.ndarray): Image to be converted to tensor.
+
+    Returns:
+        Tensor: Converted image.
+    """
```

Comment:
If we indeed only consider that this function only supports the `PIL -> tensor` conversion, then maybe a better name would be `pil_to_tensor` or something like that? Open to suggestions

Reply:
Can you take a second pass at your earliest convenience?  

One of the tests is a little awkward in that ToPILImage converts FloatTensors to bytes.  
The other thing was I'm unsure of the parameter name "swap_to_channelsfirst". 

Let me know.